packages/steveo/src/lib/context.ts

```diff
@@ -28,16 +28,15 @@ export const getDuration = (start = undefined) => {
 };
 
 export const getContext = params => {
-  const { _meta: meta, context = {} } = params;
```

Comment:
This just reverts the getContext to what it previously was. The context is part of the meta, and not part of the params.